DESIGN_DOCS.md

```diff
@@ -0,0 +1,65 @@
+# Design Documents
+
+## Purposes of this document
+* To describe what our design documents are for, their structure, and what goes in them.
+* To describe how DSNP Design Documents are produced and where they're published.
+
+## What Is a Design Document?
```

Comment:
One non-blocking comment: Do we feel we need to mention that design docs will fall under the copyright of the repository? (Currently MIT, but all code DSNP repos are Apache 2 and the whitepaper is CC-SA 4.0)

Reply:
I don't think it's necessary to mention the copyright within the Design Doc metadoc.